frontend/src/views/HomeView.vue

```diff
@@ -4,21 +4,21 @@ import InfoBox from '@/elements/home/InfoBox.vue';
 import HomeFooter from '@/components/HomeFooter.vue';
 import { useUserStore } from '@/stores/user-store';
 import { useRouter } from 'vue-router';
-import {storeToRefs} from "pinia";
+import { storeToRefs } from "pinia";
 
 const router = useRouter();
 
 const userStore = useUserStore();
 const { authenticated: isAuthenticated } = storeToRefs(userStore);
 
 const enter = () => {
-  router.push('/calendar');
+  router.push({ name: 'dashboard' });
```

Comment:
nice improvement!

Reply:
Thanks! Yes it's much easier to change/refactor/redirect routes if we use route names only. I'll check if I find some more direct urls and fix them too.